ui/components/multichain/app-footer/app-footer.js

```diff
@@ -109,16 +136,39 @@ export const AppFooter = () => {
         alignItems={AlignItems.center}
         tabIndex={0}
       >
-        <Icon
-          data-testid="app-footer-connections-button"
-          color={
-            activeConnections
-              ? IconColor.primaryDefault
-              : IconColor.iconAlternative
-          }
-          name={IconName.Global}
-          size={IconSize.Lg}
-        />
+        {connectedSite ? (
+          <BadgeWrapper
```

Comment:
I think we can wrap the BadgeWrapper in the Box and apply `alignItems={AlignItems.center}` prop to avoid css

```
          <Box alignItems={AlignItems.center}>
            <BadgeWrapper
              display={Display.Flex}
              className="app-footer__connected-badge"
              badge={
                <AvatarNetwork
                  backgroundColor={testNetworkBackgroundColor}
                  size={AvatarNetworkSize.Xs}
                  name={currentChain.nickname}
                  src={currentChain.rpcPrefs?.imageUrl}
                  borderWidth={2}
                  borderColor={BackgroundColor.backgroundDefault}
                />
              }
            >
              <AvatarFavicon
                size={Size.SM}
                src={connectedAvatar}
                name={connectedAvatarName}
              />
            </BadgeWrapper>
          </Box>
```

Reply:
YES @NidhiKJha !  Great catch!